front-end/src/components/analysis-page/IntentMenu.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import store from "../../store.js";
+import { setDisplayingQuestion } from '../../store/analyzeTranscriptSlice.js';
+
+class IntentMenu extends Component {
+    
+    constructor(props) {
+        super(props);
+        this.ref = React.createRef();
```

Comment:
Also I'm not currently sure what this does, if you are going to use a ref can you give it a more descriptive name and/or comment what it does?

Reply:
I will try redo